openquake/hazardlib/geo/surface/kite_fault.py

```diff
@@ -0,0 +1,770 @@
+# -*- coding: utf-8 -*-
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+#
+# Copyright (C) 2020 GEM Foundation
+#
+# OpenQuake is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# OpenQuake is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with OpenQuake. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Module :mod:`openquake.hazardlib.geo.surface.kite_fault` defines
+:class:`KiteSurface`.
+"""
+
+import copy
+import numpy as np
+
+from pyproj import Geod
+from openquake.hazardlib.geo.mesh import Mesh
+from openquake.hazardlib.geo import Point, Line
+from openquake.hazardlib.geo.surface.base import BaseSurface
+from openquake.hazardlib.geo.geodetic import npoints_towards
+from openquake.hazardlib.geo.geodetic import distance, azimuth
+
+TOL = 0.5
+TOLERANCE = 0.2
+
```

Comment:
What's the difference between TOL and TOLERANCE?

Reply:
Fixed